pkg/controller/networkaddonsconfig/networkaddonsconfig_controller_test.go

```diff
@@ -0,0 +1,123 @@
+package networkaddonsconfig
+
+import (
+	"fmt"
+
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/names"
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/render"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type checkUnit struct {
+	key           string
+	shouldExist   bool
+	expectedValue string
+}
+
+var _ = Describe("Networkaddonsconfig", func() {
+	Context("When CR is not labeled", func() {
+		It("Should find only default labels", func() {
+			renderData := render.MakeRenderData()
+			objs, err := render.RenderTemplate("testdata/data.yaml", &renderData)
```

Comment:
could you rename `data.yaml` to be meaningful? suggestion ``deployment-without-relation-labels.yaml"

Reply:
I think not , especially if we add data as this one suggests
https://github.com/kubevirt/cluster-network-addons-operator/pull/846#discussion_r646128959

Reply:
then could be "objects-without-relation-labels.yaml" or something the like

Reply:
The objects dont need to have relation labels, that data is raw, so i don't think we should emphasize it
its like testdata/template.yaml that already appears on other folders

i would prefer to keep it simple and raw if possible please